src/docs/environment.mdx

```diff
@@ -83,10 +85,11 @@ curl https://get.volta.sh | bash
 The volta installer will tell you to "open a new terminal to start using Volta", but you don't have to! You can just reload your shell like before:
 
 ```bash
-PATH="" exec /bin/bash -l
```

Comment:
lol rip

Reply:
yeah i figure if people actually know wtf is going on here they likely can fix it themselves (or just reopen shells)

did this to simplify things, and I think some of this is "ya'll gotta learn how shells work"

Reply:
(asside probably more simplification we can do here, still need to get collapsed blocks in tho)